actor/v7pushaction/create_deployment_for_push_plan.go

```diff
@@ -8,10 +8,15 @@ import (
 func (actor Actor) CreateDeploymentForApplication(pushPlan PushPlan, eventStream chan<- *PushEvent, progressBar ProgressBar) (PushPlan, Warnings, error) {
 	eventStream <- &PushEvent{Plan: pushPlan, Event: StartingDeployment}
 
-	var dep resources.Deployment
-	dep.DropletGUID = pushPlan.DropletGUID
-	dep.Strategy = pushPlan.Strategy
-	dep.Relationships = resources.Relationships{constant.RelationshipTypeApplication: resources.Relationship{GUID: pushPlan.Application.GUID}}
+	dep := resources.Deployment{
```

Comment:
[comment, no action] good improvement / we should be using this style. I wonder if our linter catches the assignment approach you've replaced